lib/testing_appserver/modules/people/user_creation_form.rb

```diff
@@ -35,15 +35,15 @@ def wait_to_load
       @instance.webdriver.wait_until { first_name_element.present? }
     end
 
-    def create_new_user(params = {}, generate_pwd: false)
+    def create_new_user(params = {}, generate_pwd = false)
```

Comment:
`rubocop` don't like it and I think it's a bad idea to have `params` and have `generate_pwd` key after that

I think `generate_pwd` should be part of `params`